include/dxc/dxcisense.h

```diff
@@ -785,7 +785,7 @@ IDxcTranslationUnit : public IUnknown
     _Out_ BSTR* errorMessage) = 0;
   virtual HRESULT STDMETHODCALLTYPE GetInclusionList(_Out_ unsigned* pResultCount, _Outptr_result_buffer_(*pResultCount) IDxcInclusion*** pResult) = 0;
   virtual HRESULT STDMETHODCALLTYPE CodeCompleteAt(
-      _In_ char *fileName, unsigned line, unsigned column,
+      _In_ const char *fileName, unsigned line, unsigned column,
```

Comment:
Need people to review this. It's a change to the interface, add a const to parameter.

Reply:
I think it is a good change, and we only have one active user of this interface that I know of.